src/app/modules/learning/unit/unit-item/unit-item.component.html

```diff
@@ -39,9 +39,9 @@
         <mat-icon matTooltip="Zadatak" style="margin-top: 4px; min-width: 24px;" [ngClass]="{ 'done': item.isSatisfied}">rebase_edit</mat-icon>
         <span style="flex-grow: 1;" [ngClass]="{ 'done': item.isSatisfied}">{{item.task.name}}</span>
         <span style="min-width: 48px; text-align: right;">
-          <mat-icon *ngIf="item.isSatisfied" color="primary" style="margin-top: 4px;" matTooltip="Rešeno!">check_circle</mat-icon>
+          <mat-icon *ngIf="item.isSatisfied" color="primary" style="margin-top: 4px;" matTooltip="Ocenjeno!">check_circle</mat-icon>
           <span *ngIf="!item.isSatisfied">
-            0%
+            {{item.task.completedSteps/item.task.totalSteps * 100}}%
```

Comment:
Are we talking about `completedSteps` or `gradedSteps` here? It seems that `isSatisfied` is assigned to true if the task is `graded`, but the percentile is related to `completed`.

I would say that `isSatisfied` is true if it is completed or graded, and I would also expand the `TaskProgressSummary` to include the won points and max points, which can be displayed left of the checkmark if they exist (if the task is graded).